lms/djangoapps/certificates/views/webview.py

```diff
@@ -235,20 +235,7 @@ def _update_course_context(request, context, course, course_key, platform_name):
     context['accomplishment_copy_course_name'] = accomplishment_copy_course_name
     course_number = course.display_coursenumber if course.display_coursenumber else course.number
     context['course_number'] = course_number
-    if context['organization_long_name']:
-        # Translators:  This text represents the description of course
-        context['accomplishment_copy_course_description'] = _(u'a course of study offered by {partner_short_name}, '
-                                                              'an online learning initiative of '
-                                                              '{partner_long_name}.').format(
-            partner_short_name=context['organization_short_name'],
-            partner_long_name=context['organization_long_name'],
-            platform_name=platform_name)
-    else:
-        # Translators:  This text represents the description of course
-        context['accomplishment_copy_course_description'] = _('a course of study offered by '
-                                                              '{partner_short_name}.').format(
-            partner_short_name=context['organization_short_name'],
-            platform_name=platform_name)
+    context['accomplishment_copy_course_description'] = context['certificate_data'].get('course_description', '')
```

Comment:
The same comment as https://github.com/openedx/edx-platform/pull/29850/files#r796368652

Reply:
Made the changes.